src/data/docs/language-reference/README.md

```diff
@@ -272,7 +272,7 @@ Any identifier, including a namespace-qualified one, can appear _hash-qualified_
 
 #### Reserved words
 
-The following names are reserved by Unison and cannot be used as identifiers: `=`, `:`, `->`, `if`, `then`, `else`, `forall`, `handle`, `in`, `unique`, `where`, `use`, `and`, `or`, `true`, `false`, `type`, `ability`, `alias`, `let`, `namespace`, `case`, `of`, `with`.
+The following names are reserved by Unison and cannot be used as identifiers: `=`, `:`, `->`, `if`, `then`, `else`, `forall`, `handle`, `unique`, `where`, `use`, `and`, `or`, `true`, `false`, `type`, `ability`, `alias`, `let`, `namespace`, `case`, `of`, `with`.
```

Comment:
Should "in" still be reserved? ... could make it easier to add syntax like Haskell's "let .. in .." later. I noticed that "with" was already in this list of reserved words.

Reply:
I don’t think we’re gonna want `let ... in ...`, I would say the fewer reserved words, the better.  However https://github.com/unisonweb/unison/issues/742 will make that less important.